multidict/_compat.py

```diff
@@ -6,4 +6,4 @@
 
 PYPY = platform.python_implementation() == 'PyPy'
 
-USE_CYTHON_EXTENSIONS = USE_CYTHON = not NO_EXTENSIONS and not PYPY
+USE_C_EXTENSIONS = not NO_EXTENSIONS and not PYPY
```

Comment:
Public API change, needs a deprecation warning.

Reply:
What I should do? Can you show an example?

Reply:
one of the ways is to declare `__getattribute__(name)` function but that'd work under Python 3.7 only. the other way is to have a hack with replacing the module namespace with a class instance.